_episodes/01-introduction.md

```diff
@@ -103,22 +91,18 @@ two specific perspectives have primarily influenced Software and Data Carpentry'
 One perspective is *cognitivism*,
 which treats learning as a problem in neuropsychology.
 Cognitivists focus their attention on things like pattern recognition,
-memory formation,
-and recall.
-It is good at answering low-level questions,
-but generally ignores larger issues like, "What do we mean by 'learning'?"
-and, "Who gets to decide?"
```

Comment:
I think this sentence is important, especially given MOOCs and ongoing attacks on (higher) education.